index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <style>
+  @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap');
+      </style>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"/>
+</head>
+<body>
+    <section class="navigation">
+        <nav class="header">
+            <p class="logo">Josiah</p>
+            <a href="#" class="menu"><img src="img/menu.png" alt=""></a>
+        </nav>
+    </section>
+
+    <section class="bio-section">
+        <div class="bio">
+            <h3 class="bio-header">I'm Josiah <br>Glad to see you</h3>
+            <p class="bio-text">I'm a software developer! I can help ou build a product, feature or website. Look Through
+            <br>some of my work and experience! If you like what you see and have a project<br> you need coded, don't hesitate to contact me.</p>
+            <p class="connect"><a href="#" class="connect-link">lets connect</a></p>
+        </div>
+        <div class="connect-icons">
+            <i class="fa-brands fa-twitter"></i>
+            <i class="fa-brands fa-linkedin"></i>
+            <i class="fa-brands fa-facebook"></i>
+            <i class="fa-brands fa-github"></i>
+            <i class="fa-brands fa-angellist"></i>
+
+
+        </div>
```

Comment:
Please consider putting these icons in a list as it is more semantic and consider putting each icon into a link tag as this will enable you to link the icons to your corresponding socials. In the end, the list might look something like this: 
```
<ul class="connect-icons">
            <li><a href="social_link"><i class="fa-brands fa-twitter"></i></a></li>
            <li><a href="social_link"><i class="fa-brands fa-linkedin"></i></a></li>
            <li><a href="social_link"><i class="fa-brands fa-facebook"></i></a></li>
            <li><a href="social_link"><i class="fa-brands fa-github"></i></a></li>
            <li><a href="social_link"><i class="fa-brands fa-angellist"></i></a></li>
</ul>
```